packages/server/tests/acceptance.spec.ts

```diff
@@ -185,384 +229,223 @@ describe('RPC Server Integration Tests', async function () {
-        callUnsupportedRelayMethod(this.relayClient, 'eth_getWork', []);
+        utils.callUnsupportedRelayMethod(this.relayClient, 'eth_getWork', []);
     });
 
     it('should execute "eth_hashrate"', async function () {
-        const res = await callSupportedRelayMethod(this.relayClient, 'eth_hashrate', []);
+        const res = await utils.callSupportedRelayMethod(this.relayClient, 'eth_hashrate', []);
         expect(res.data.result).to.be.equal('0x0');
     });
 
     it('should execute "eth_mining"', async function () {
-        const res = await callSupportedRelayMethod(this.relayClient, 'eth_mining', []);
+        const res = await utils.callSupportedRelayMethod(this.relayClient, 'eth_mining', []);
         expect(res.data.result).to.be.equal(false);
     });
 
     it('should execute "eth_submitWork"', async function () {
-        const res = await callSupportedRelayMethod(this.relayClient, 'eth_submitWork', []);
+        const res = await utils.callSupportedRelayMethod(this.relayClient, 'eth_submitWork', []);
         expect(res.data.result).to.be.equal(false);
     });
 
-    it('should execute eth_sendRawTransaction legacy', async function () {
-        const res = await callSupportedRelayMethod(this.relayClient, 'eth_sendRawTransaction', ['0x' + legacyTransactionHex]);
-        expect(res.data.result).to.be.equal('0x9ffbd69c44cf643ed8d1e756b505e545e3b5dd3a6b5ef9da1d8eca6679706594');
+    it('should execute "eth_sendRawTransaction" for legacy transactions', async function () {
+        const signedTx = await utils.signRawTransaction({
+            ...defaultLegacyTransactionData,
+            to: mirrorContract.evm_address
+        }, ethCompPrivateKey3);
+
+        const res = await utils.callSupportedRelayMethod(this.relayClient, 'eth_sendRawTransaction', [signedTx]);
+        expect(res.data.result).to.be.equal('0x93c4b87f7fe3d6071a9c58acf5b64ec976c60ca2017f21fac42f445472885727');
     });
 
-    it('should execute "eth_sendRawTransaction" london', async function () {
-        const res = await callSupportedRelayMethod(this.relayClient, 'eth_sendRawTransaction', ['0x' + londonTransactionHex]);
-        expect(res.data.result).to.be.equal('0xcdbbfb6400aab319f97d32c38e285f0d0399c2b48b683f04878b5f07eb0d50e3');
+    it('should fail "eth_sendRawTransaction" for Legacy 2930 transactions', async function () {
+        // INVALID_ETHEREUM_TX
+        const signedTx = await utils.signRawTransaction({
+            ...defaultLegacy2930TransactionData,
+            to: mirrorContract.evm_address,
+            nonce: 1
+        }, ethCompPrivateKey3);
+
+        const res = await utils.callFailingRelayMethod(this.relayClient, 'eth_sendRawTransaction', [signedTx]);
+        expect(res.data.error.message).to.be.equal('Internal error');
+        expect(res.data.error.code).to.be.equal(-32603);
+    });
+
+    it('should execute "eth_sendRawTransaction" for London transactions', async function () {
```

Comment:
I think that it would be best to assert that actually the sender got his balanced reduced and the `to` address got its balance increased with `1`. This would verify that the `sendRawTransaction` was processed, it did not error and returned the manually computed eth hash